neo/Ledger/MemoryPool.cs

```diff
@@ -279,7 +279,7 @@ private PoolItem GetLowestFeeTransaction(out Dictionary<UInt256, PoolItem> unsor
 
             if (minItem != null)
             {
-                unsortedTxPool = sortedPool == _unverifiedSortedLowPriorityTransactions
+                unsortedTxPool = minItem.Transaction.IsLowPriority
```

Comment:
Just make it use Object.ReferenceEquals

Reply:
Just as fast and more clear IMO

Reply:
Let's try

Reply:
Done, Jeff, it works as expected.
It is great that you did the UT.

Reply:
Thanks for the tip.